terraform/aws/components/nhais_responder/locals_env_variables.tf

```diff
@@ -0,0 +1,64 @@
+locals {
+  environment_variables = concat(var.nhais_environment_variables,[
+    {
+      name  = "NHAIS_RESPONDER_OUTBOUND_SERVER_PORT"
```

Comment:
`NHAIS_OUTBOUND_SERVER_PORT`

NHAIS_RESPONDER... prefix is not used any any of the environment variable names